ovos_config/base.py

```diff
@@ -1,272 +1,3 @@
-
-    def merge(self, *args, **kwargs):
-        if not self.allow_overwrite:
-            raise PermissionError(f"{self.path} is read only! it can not be modified at runtime")
-        super().merge(*args, **kwargs)
-
-    def store(self, path=None):
-        if not self.allow_overwrite:
-            raise PermissionError(f"{self.path} is read only! it can not be modified at runtime")
-        super().store(path)
-
-
-class MycroftDefaultConfig(ReadOnlyConfig):
-    def __init__(self):
-        super().__init__(DEFAULT_CONFIG)
-
-    def set_root_config_path(self, root_config):
-        # in case we got it wrong / non standard
-        self.path = root_config
-        self.reload()
-
-
-class MycroftSystemConfig(ReadOnlyConfig):
-    def __init__(self, allow_overwrite=False):
-        super().__init__(SYSTEM_CONFIG, allow_overwrite)
-
-
-class RemoteConf(LocalConf):
-    """Config dictionary fetched from mycroft.ai."""
-
-    def __init__(self, cache=WEB_CONFIG_CACHE):
-        super(RemoteConf, self).__init__(cache)
-
-    def reload(self):
-        if selene_api is None:
-            self.load_local(self.path)
-            return
-        try:
-            if not selene_api.is_paired():
-                self.load_local(self.path)
-                return
-
-            if selene_api.is_backend_disabled():
-                # disable options that require backend
-                config = {
-                    "server": {
-                        "metrics": False,
-                        "sync_skill_settings": False
-                    },
-                    "skills": {"upload_skill_manifest": False},
-                    "opt_in": False
-                }
-                for key in config:
-                    self.__setitem__(key, config[key])
-            else:
-                api = selene_api.DeviceApi()
-                setting = api.get_settings()
-                location = None
-                try:
-                    location = api.get_location()
-                except Exception as e:
-                    LOG.error(f"Exception fetching remote location: {e}")
-                    if exists(self.path) and isfile(self.path):
-                        location = load_commented_json(self.path).get('location')
-
-                if location:
-                    setting["location"] = location
-                # Remove server specific entries
-                config = {}
-                translate_remote(config, setting)
-
-                for key in config:
-                    self.__setitem__(key, config[key])
-                self.store(self.path)
-
-        except Exception as e:
-            LOG.error(f"Exception fetching remote configuration: {e}")
-            self.load_local(self.path)
-
-
-class MycroftUserConfig(LocalConf):
-    def __init__(self):
-        super().__init__(USER_CONFIG)
-
-
-MycroftXDGConfig = MycroftUserConfig
+LOG.warning("This reference is deprecated, import from ovos_config.models")
```

Comment:
This file could be removed, it was added in previous pr only and that is a buggy release to be yanked

Reply:
Ah, right. I'll just remove it entirely